scripts/copy-and-verify-results.mjs

```diff
@@ -67,7 +67,7 @@ function compareResultsAgainstGoldens(library) {
   }
   // A constant, to make sure that if we add/remove any tests, that we add that test to all
   // tested libraries.
-  const numberOfTests = 32;
+  const numberOfTests = 34;
```

Comment:
Do you know why this is adding two tests? It looks like there's just "will not overwrite unwriteable properties"

Reply:
Oh, it's chrome + firefox counted separately